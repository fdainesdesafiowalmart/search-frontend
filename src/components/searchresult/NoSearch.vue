<template>
    <div class="no-search-container">
        <div class="no-search-body">
            <div class="no-search-description">
                Ingrese un patrón de búsqueda
            </div>
        </div>
    </div>
</template>

<script>
export default {
  name: 'NoSearch'
}
</script>

<style scoped>
.no-search-container {
    width: 100%;
    background-color: rgb(255, 255, 255);
    color: rgb(66, 66, 66);
    font-size: 12px;
    box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 13px 0px, rgba(0, 0, 0, 0.07) 0px 1px 3px 0px;
    border-radius: 4px;
    padding: 15px;
    margin: 10px 5px 25px 10px;
}

.no-search-body {
    text-align: center;
    margin-bottom: 30px;
    background-color: rgba(0, 113, 206, 0.08);
    padding-top: 10px;
    border-radius: 8px;
}

.no-search-description {
    font-size: 16px;
    color: rgb(65, 64, 66);
    padding: 0px 15px 20px;
}

</style>
