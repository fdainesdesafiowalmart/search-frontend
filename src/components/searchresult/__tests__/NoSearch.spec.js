import { shallowMount, RouterLinkStub } from '@vue/test-utils'

import NoSearch from '../NoSearch.vue'

describe('NoSearch.vue', () => {
  it('its a Vue instance', () => {
    const wrapper = shallowMount(NoSearch, {
      stubs: {
        RouterLink: RouterLinkStub
      }
    })

    expect(wrapper.vm).toBeTruthy();
  })

  it('renders static content as expected', () => {
    const wrapper = shallowMount(NoSearch, {
      stubs: {
        RouterLink: RouterLinkStub
      }
    })

    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})