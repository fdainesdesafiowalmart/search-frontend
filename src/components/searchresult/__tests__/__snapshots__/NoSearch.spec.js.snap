// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoSearch.vue renders static content as expected 1`] = `
<div
  class="no-search-container"
>
  <div
    class="no-search-body"
  >
    <div
      class="no-search-description"
    >
      
            Ingrese un patrón de búsqueda
        
    </div>
  </div>
</div>
`;
